gitman/git.py

```diff
@@ -16,49 +16,67 @@ def git(*args, **kwargs):
     return call('git', *args, **kwargs)
 
 
+def gitsvn(*args, **kwargs):
+    return call('git', 'svn', *args, **kwargs)
+
+
+def in_svn_repo():
+    """Determine if the current working directory is inside a subversion repository."""
+    output = git('show', '--shortstat', _show=False, _ignore=True)
+    if any('git-svn-id' in line for line in output):
```

Comment:
This could be condensed to one line: `return any('git-svn-id' in line for line in output)`

Reply:
Sure... np...